elastic/logs/README.md

```diff
@@ -291,27 +325,21 @@ This challenge executes indexing and querying concurrently. Queries will be issu
 
 Note: If the indexing load is higher than the cluster can support, a time lag will start to occur on the indexed documents. This may result in queries returning with no hits as the expected data has yet to be indexed.
 
-### Many Shards Basic (many-shards-base)
-
-This challenge measures performance (throughput) with big (and increasing) number of indices. It sets up initial set
-of indices (count controlled by `data.initial.indices` param) with `auditbeat` template and ILM policy (hot tier only),
-optional initial set of indices that will go to frozen tier through ILM searchable snapshot action
-(count controlled by `data.initial.frozen.indices` param) and then index to small set of data streams. These data streams
-will rollover every 10m
+### Many Shards Snapshots (many-shards-snapshots)
 
-### Many Shards Full (many-shards-full)
+This benchmarks aims to evaluate the performance of the Log Monitoring part of Elastic's Observability solution with a large amount of shards. It sets up initial set of indices (count controlled by `data.initial.indices` param) with a large amount of shards, with `auditbeat` template and ILM policy (hot tier only) and then sequentially takes a configurable via `snapshot_counts` number of snapshots. These data streams will almost never rollover (rollover based on size with 100gb as max_size). Used for benchmarks to help identify regressions related to snapshots with high index counts. The performance can be evaluated by the `service_time` of the `wait-for-snapshots` task.
 
-Same as above but additionally it sets up SLM with snapshotting every index every 15min
+Note that this challenge requires you to be able to successfully create a snapshot repository using the `snapshot_repo_type`, and `many_shards_snapshot_repo_settings` track parameters.
 
 ### Many Shards Quantitative (many-shards-quantitative)
 
 This challenge aims to get more specific numbers of what we can support in terms of indices count. It creates initial 
 set of indices as before and then index to small set of data streams. These data streams will almost never
-rollover (rollover based on size with 150gb as `max_size`). This is supposed to be run with multiple values of
```

Comment:
both many-shards use 100gb as a rollover value https://github.com/elastic/rally-tracks/blob/5f706b5d0c05d9ad070150b691e10d309bcac3ab/elastic/logs/ilm/many-shards-quantitative/auditbeat-quantitative.json#L8
Was the intent 150gb, should we update the docs or the policy?

Reply:
I think we are happy with the `100gb` we have now. @original-brownbear WDYT?

Reply:
Yea I don't think it makes a difference, we almost index nothing per index :)

Reply:
sounds good! i will update the docs then just to be consistent :)